packages/datagateway-dataview/src/views/detailsPanels/dls/datasetDetailsPanel.component.tsx

```diff
@@ -52,8 +52,10 @@ const DatasetDetailsPanel = (
   }, [value, detailsPanelResize]);
 
   return (
-    <div id="details-panel">
+    <div id="details-panel" style={{ minWidth: 0 }}>
       <Tabs
+        variant="scrollable"
+        scrollButtons="auto"
```

Comment:
Think I've got the text wrapping working for the details panel.

As for the missing publications tab, for xs (<600px) screens the scroll buttons do not appear and the missing tabs will be cut off. You can still access the tabs by shift+scrolling on the element (or swiping on mobile, which is what the xs screen is intended for). Unfortunately the tab element from MUI is a bit limited in the scrolling options, the only other way to enforce scroll buttons at xs would be to always have them present at all screen sizes whether scrolling is needed or not. I'm assuming most people will be on larger screens so didn't want to do this as it makes the tabs look misaligned in the cases where the buttons aren't needed. Before this change the scroll buttons didn't appear at all, and it wouldn't have been possible to shift+scroll to the other tabs.

I'd rather leave this as it is, but if it turns out there's a demand for scroll buttons on xs screens it can be changed at a later date?

There was also the same issue as for the text wrapping, where because of the minimum element width the right scroll button was getting cut off between 600 and 720 px wide screens but that should be fixed now as well.

Reply:
Sorry, I am not sure if I was clear enough but this is to do with the **table view** not the card view. While the card view wraps the text nicely and it is possible to see the _Publication_ tab when scrolling the tabs, the table view does not wrap the text and the _Publication_ tab is missing. I now that the missing tabs are cut off on small screens so scrolling of tabs is required to be able to see all the tabs however the _Publication_ tab is missing even when I scroll the tabs all the way to the end, see the GIF below. I also just noticed that the _Results_ label at the top of the screen is cutting off too on xs screens.
![datagateway-dataview](https://user-images.githubusercontent.com/45173816/103662549-feccea00-4f67-11eb-809c-0842e904c6d8.gif)

Reply:
That's really strange, I can't replicate what the gif is showing locally. I'm doing this on the DLS visits table since that looked like where you were? The only thing I can think of would be if the changes to common haven't been compiled. Are there any errors when running `yarn tsc` in `datagateway-common` after pulling the latest commit?

Sorry for overexplaining how the tab scrolling works, it confused me a bit when I first read about them and like you said I wasn't totally sure where the problem was so thought it might be that. As for the results I'll see about stopping it from clipping.

Reply:
Oh I see what happened here - you are right, the changes to common have not compiled. Thanks for pointing this out, it all looks good now.